### ISSUE TITLE
EGL's game update and verify action not working

### ISSUE THREAD

**Issue by menguele** [Heroic-Games-Launcher/HeroicGamesLauncher#2973]
### Describe the bug

Trying to update, in this case, Fortnite, results in a error, saying soime files are corrupt, trying to verify it results in another error.

### Add logs

```text
An error has occurred! Try to Logout and Login on your Epic account.
=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 75% (949.0/1261.4 MiB) [373.3 MiB/s]=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 76% (961.0/1261.4 MiB) [373.9 MiB/s]=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 77% (973.0/1261.4 MiB) [374.1 MiB/s]=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 78% (985.0/1261.4 MiB) [374.6 MiB/s]=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 79% (997.0/1261.4 MiB) [375.0 MiB/s]=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 80% (1009.0/1261.4 MiB) [375.4 MiB/s]=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 81% (1021.0/1261.4 MiB) [375.0 MiB/s]=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 82% (1033.0/1261.4 MiB) [375.3 MiB/s]=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 83% (1045.0/1261.4 MiB) [375.5 MiB/s]=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 84% (1057.0/1261.4 MiB) [375.8 MiB/s]=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 85% (1069.0/1261.4 MiB) [376.1 MiB/s]=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 86% (1081.0/1261.4 MiB) [376.4 MiB/s]=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 87% (1093.0/1261.4 MiB) [376.5 MiB/s]=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 88% (1105.0/1261.4 MiB) [376.8 MiB/s]=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 89% (1117.0/1261.4 MiB) [377.0 MiB/s]=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 90% (1129.0/1261.4 MiB) [377.2 MiB/s]=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 90% (1141.0/1261.4 MiB) [377.0 MiB/s]=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 91% (1153.0/1261.4 MiB) [377.3 MiB/s]=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 92% (1165.0/1261.4 MiB) [377.3 MiB/s]=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 93% (1177.0/1261.4 MiB) [377.3 MiB/s]=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 94% (1189.0/1261.4 MiB) [377.3 MiB/s]=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 95% (1201.0/1261.4 MiB) [377.1 MiB/s]=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 96% (1213.0/1261.4 MiB) [377.2 MiB/s]=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 97% (1225.0/1261.4 MiB) [377.4 MiB/s]=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 98% (1237.0/1261.4 MiB) [377.7 MiB/s]=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 99% (1249.0/1261.4 MiB) [377.9 MiB/s]=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 100% (1261.0/1261.4 MiB) [377.8 MiB/s]=> Verifying large file "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas": 100% (1261.4/1261.4 MiB) [377.8 MiB/s]
Verification progress: 533/587 (44.0%) [376.9 MiB/s]Verification progress: 534/587 (44.0%) [376.9 MiB/s]Verification progress: 535/587 (44.0%) [376.9 MiB/s]
Verification progress: 536/587 (44.0%) [376.9 MiB/s]
Verification progress: 537/587 (44.0%) [376.9 MiB/s]
Verification progress: 538/587 (44.0%) [376.9 MiB/s]Verification progress: 539/587 (44.0%) [376.9 MiB/s]
Verification progress: 540/587 (44.0%) [376.9 MiB/s]
Verification progress: 541/587 (44.0%) [376.9 MiB/s]
Verification progress: 542/587 (44.0%) [376.9 MiB/s]
Verification progress: 543/587 (44.0%) [376.9 MiB/s]
Verification progress: 544/587 (44.0%) [376.9 MiB/s]Verification progress: 545/587 (44.0%) [376.9 MiB/s]
Verification progress: 546/587 (44.0%) [376.9 MiB/s]
Verification progress: 547/587 (44.0%) [376.9 MiB/s]
Verification progress: 548/587 (44.0%) [376.9 MiB/s]
Verification progress: 549/587 (44.0%) [376.9 MiB/s]
Verification progress: 550/587 (44.0%) [376.9 MiB/s]Verification progress: 551/587 (44.0%) [376.9 MiB/s]Verification progress: 552/587 (44.0%) [376.9 MiB/s]Verification progress: 553/587 (44.0%) [376.9 MiB/s]Verification progress: 554/587 (44.0%) [376.9 MiB/s]Verification progress: 555/587 (44.0%) [376.9 MiB/s]Verification progress: 556/587 (44.0%) [376.9 MiB/s]Verification progress: 557/587 (44.0%) [376.9 MiB/s]Verification progress: 558/587 (44.0%) [376.9 MiB/s]Verification progress: 559/587 (44.0%) [376.9 MiB/s]
Verification progress: 560/587 (44.0%) [376.9 MiB/s]
Verification progress: 561/587 (44.0%) [376.9 MiB/s]
Verification progress: 562/587 (44.0%) [376.9 MiB/s]Verification progress: 563/587 (44.0%) [376.9 MiB/s]
Verification progress: 564/587 (44.0%) [376.9 MiB/s]
Verification progress: 565/587 (44.0%) [376.9 MiB/s]
Verification progress: 566/587 (44.0%) [376.9 MiB/s]
Verification progress: 567/587 (44.0%) [376.9 MiB/s]
Verification progress: 568/587 (44.0%) [376.9 MiB/s]Verification progress: 569/587 (44.0%) [376.9 MiB/s]
Verification progress: 570/587 (44.0%) [376.9 MiB/s]
Verification progress: 571/587 (44.0%) [376.9 MiB/s]
Verification progress: 572/587 (44.0%) [376.9 MiB/s]
Verification progress: 573/587 (44.0%) [376.9 MiB/s]
Verification progress: 574/587 (44.0%) [376.9 MiB/s]
Verification progress: 575/587 (44.0%) [376.9 MiB/s]
Verification progress: 576/587 (44.0%) [376.9 MiB/s]
Verification progress: 577/587 (44.0%) [376.9 MiB/s]Verification progress: 578/587 (44.0%) [376.9 MiB/s]
Verification progress: 579/587 (44.0%) [376.9 MiB/s]
Verification progress: 580/587 (44.0%) [376.9 MiB/s]
Verification progress: 581/587 (44.0%) [376.9 MiB/s]
Verification progress: 582/587 (44.0%) [376.9 MiB/s]Verification progress: 583/587 (44.1%) [376.9 MiB/s]Verification progress: 584/587 (44.1%) [376.9 MiB/s]Verification progress: 585/587 (44.2%) [58.5 MiB/s]Verification progress: 586/587 (44.2%) [58.5 MiB/s]Verification progress: 587/587 (44.2%) [58.5 MiB/s]Verification progress: 587/587 (44.2%) [58.5 MiB/s]
You are about to install Fortnite, this application supports selective downloads.
The following optional packs are available (tag - name):
* hd_textures - High Resolution Textures
* dx12 - DirectX 12 Shaders
* ondemand - Disable Cosmetic Streaming
* stw - Fortnite Save the World
* stw_hd_textures - Fortnite Save the World High Resolution Textures
* lang_cn -[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakchunk1007optional-WindowsClient.utoc"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakchunk1008-WindowsClient.pak"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakchunk1008-WindowsClient.sig"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakchunk1008-WindowsClient.ucas"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakchunk1008-WindowsClient.utoc"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk1009-WindowsClient.pak"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk1009-WindowsClient.sig"[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk1009-WindowsClient.ucas"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk1009-WindowsClient.utoc"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk1010-WindowsClient.pak"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk1010-WindowsClient.sig"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk1010-WindowsClient.ucas"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk1010-WindowsClient.utoc"[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk1010optional-WindowsClient.pak"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk1010optional-WindowsClient.sig"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk1010optional-WindowsClient.ucas"[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk1010optional-WindowsClient.utoc"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk1011-WindowsClient.pak"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk1011-WindowsClient.sig"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk1011-WindowsClient.ucas"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk1011-WindowsClient.utoc"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk1012-WindowsClient.pak"[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk1012-WindowsClient.sig"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk1012-WindowsClient.ucas"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk1012-WindowsClient.utoc"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk1012optional-WindowsClient.pak"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk1012optional-WindowsClient.sig"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk1012optional-WindowsClient.ucas"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk1012optional-WindowsClient.utoc"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk10ondemand-WindowsClient.pak"[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk10ondemand-WindowsClient.sig"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk10ondemand-WindowsClient.ucas"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk10ondemand-WindowsClient.utoc"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk10ondemand-WindowsClient_s1.ucas"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk10ondemandoptional-WindowsClient.pak"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk10ondemandoptional-WindowsClient.sig"[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk10ondemandoptional-WindowsClient.ucas"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk10ondemandoptional-WindowsClient.utoc"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk10ondemandoptional-WindowsClient_s1.ucas"[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk10ondemandoptional-WindowsClient_s2.ucas"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk10ondemandoptional-WindowsClient_s3.ucas"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk10ondemandoptional-WindowsClient_s4.ucas"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk10optional-WindowsClient.pak"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk10optional-WindowsClient.sig"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk10optional-WindowsClient.ucas"[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk10optional-WindowsClient.utoc"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk10optional-WindowsClient_s1.ucas"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakchunk10sm6-WindowsClient.pak"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakchunk10sm6-WindowsClient.sig"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakchunk10sm6-WindowsClient.ucas"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakchunk10sm6-WindowsClient.utoc"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakchunk11-WindowsClient.pak"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakchunk11-WindowsClient.sig"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakchunk11-WindowsClient.ucas"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakchunk11-WindowsClient.utoc"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk11optional-WindowsClient.pak"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk11optional-WindowsClient.sig"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk11optional-WindowsClient.ucas"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk11optional-WindowsClient.utoc"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakchunk19632-WindowsClient.pak"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakchunk19632-WindowsClient.sig"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakchunk19632-WindowsClient.utoc"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk2-WindowsClient.pak"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk2-WindowsClient.sig"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk2-WindowsClient.ucas"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk2-WindowsClient.utoc"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakchunk20-WindowsClient.pak"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakchunk20-WindowsClient.sig"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakchunk20-WindowsClient.ucas"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakchunk20-WindowsClient.utoc"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk20optional-WindowsClient.pak"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk20optional-WindowsClient.sig"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk20optional-WindowsClient.ucas"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk20optional-WindowsClient.utoc"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk2optional-WindowsClient.pak"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk2optional-WindowsClient.sig"[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk2optional-WindowsClient.ucas"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk2optional-WindowsClient.utoc"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk5-WindowsClient.pak"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk5-WindowsClient.sig"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk5-WindowsClient.ucas"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk5-WindowsClient.utoc"[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk5optional-WindowsClient.pak"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk5optional-WindowsClient.sig"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk5optional-WindowsClient.ucas"[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk5optional-WindowsClient.utoc"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakchunk6600-WindowsClient.pak"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakchunk6600-WindowsClient.sig"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakchunk6600-WindowsClient.utoc"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakchunk6601-WindowsClient.pak"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakchunk6601-WindowsClient.sig"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakchunk6601-WindowsClient.ucas"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakchunk6601-WindowsClient.utoc"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk7-WindowsClient.pak"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk7-WindowsClient.sig"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk7-WindowsClient.ucas"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk7-WindowsClient.utoc"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk7optional-WindowsClient.pak"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk7optional-WindowsClient.sig"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk7optional-WindowsClient.ucas"[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk7optional-WindowsClient.utoc"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk8-WindowsClient.pak"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk8-WindowsClient.sig"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk8-WindowsClient.ucas"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk8-WindowsClient.utoc"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk8optional-WindowsClient.pak"[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk8optional-WindowsClient.sig"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk8optional-WindowsClient.ucas"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk8optional-WindowsClient.utoc"[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk9-WindowsClient.pak"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk9-WindowsClient.sig"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk9-WindowsClient.ucas"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk9-WindowsClient.utoc"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk9optional-WindowsClient.pak"[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk9optional-WindowsClient.sig"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk9optional-WindowsClient.ucas"
[cli] ERROR: File is missing: "FortniteGame/Content/Paks/pakchunk9optional-WindowsClient.utoc"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakChunkEarly-WindowsClient.pak"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakChunkEarly-WindowsClient.sig"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakChunkEarly-WindowsClient.ucas"[cli] ERROR: File does not match hash: "FortniteGame/Content/Paks/pakChunkEarly-WindowsClient.utoc"[cli] ERROR: Verification failed, 233 file(s) corrupted, 110 file(s) are missing.Traceback (most recent call last):
File "legendary\cli.py", line 3069, in <module>File "legendary\cli.py", line 2986, in main
File "legendary\cli.py", line 923, in install_game
File "legendary\utils\cli.py", line 56, in sdl_promptFile "encodings\cp1252.py", line 19, in encode
UnicodeEncodeError: 'charmap' codec can't encode characters in position 16-21: character maps to <undefined>
[3484] Failed to execute script 'cli' due to unhandled exception!




An error has occurred! Try to Logout and Login on your Epic account.
You are about to install Fortnite, this application supports selective downloads.
The following optional packs are available (tag - name):
* dx12 - DirectX 12 Shaders
* hd_textures - High Resolution Textures
* lang_cn -[cli] INFO: Game needs to be verified before updating, switching to repair mode...[Core] INFO: Trying to re-use existing login session...[cli] INFO: Using existing repair file: B:\joseg\AppData\Roaming\heroic\legendaryConfig\legendary\tmp\Fortnite.repairTraceback (most recent call last):
File "legendary\cli.py", line 3069, in <module>
File "legendary\cli.py", line 2986, in main
File "legendary\cli.py", line 923, in install_gameFile "legendary\utils\cli.py", line 56, in sdl_prompt
File "encodings\cp1252.py", line 19, in encode
UnicodeEncodeError: 'charmap' codec can't encode characters in position 16-21: character maps to <undefined>
[8584] Failed to execute script 'cli' due to unhandled exception!
```

### Steps to reproduce

1. Click on Force update if available
2. See 1° Error
3. Click on Verify and Repair
4. See 2° error

### Expected behavior

Updating it without errorring or verifying it correctly

### Heroic Version

Latest Stable

### System Information

- Windows 10 Pro 22H2
- OS Build: 19045.2364

### Additional information

(evil) EGL isn't installed.

**Comment by Mann-Servante** [comment]
I was trying to google to find if there was a solution to this issue. I eventually gave up and updated with the epic launcher, but I was getting the same error about trying to install fortnite.
And I don't know if the verify and repair option did anything when I clicked it.

**Comment by arielj** [comment]
This should be fixed in Heroic 2.10.0
